VersionControlUI/src/DiffManager.h

```diff
@@ -57,9 +57,17 @@ class DiffComparisonPair;
 class VERSIONCONTROLUI_API DiffManager
 {
 	public:
+		enum NameChangeVisualization {
+				Summary = 0x1,
+				NameText = 0x2,
+				References = 0x4
+		};
+		using NameChangeVisualizations = QFlags<NameChangeVisualization>;
+
 		DiffManager(QString project,
 						QList<Model::SymbolMatcher> contextUnitMatcherPriorityList,
-						Model::NodeIdType targetNodeID=Model::NodeIdType{});
+						Model::NodeIdType targetNodeID=Model::NodeIdType{},
```

Comment:
you can use simply `{}` here, without the `Model::NodeIdType` after the `=`. It's a bit clearer.